### ISSUE TITLE
Visual glitch when renaming group from right-click

### ISSUE THREAD

**Issue by ProMcTagonist** [qTox/qTox#1330]
![group rename glitch](https://cloud.githubusercontent.com/assets/4358079/6477544/9036e158-c1d9-11e4-901c-1b5bf30811a4.png)

Reproduce:
1. Start or join group
2. Deselect the group
3. Right click > Set title...
4. Rename group

It disappears when the mouse moves, or when it gets moused over again if the cursor was moved away while the rename dialog was open.

**Comment by kushagrasurana** [comment]
Unable to reproduce on my ubuntu. is it fixed?

**Comment by ProMcTagonist** [comment]
Still happening to me on a fresh git pull.

**Comment by agilob** [comment]
Same here, but it looks a bit differently
![qtoxgroupchat](https://cloud.githubusercontent.com/assets/2544251/6546151/f565a798-c59f-11e4-884c-eb8815c203e9.png)

**Comment by zetok** [comment]
@agilob it looks differently because your contact list isn't big enough to show a scrollbar, am I right?

**Comment by dubslow** [comment]
Also unable to reproduce

**Comment by zetok** [comment]
@dubslow right. well, lets see differences

@ProMcTagonist @kushagrasurana @agilob @dubslow mind filling relevant part of https://github.com/tux3/qTox/wiki/Useful-bug-reports#template ?

qTox: 2ea3d73756b5e846dac461f9762eaca12dad8134

```
Info:
OS: Gentoo Linux x86_64
Kernel: 3.18.7
Xorg: 1.16.4
nouveau: 1.0.11
Qt: 5.4.1
DE/WM: i3

all compiled with
GCC: gcc version 4.8.3 (Gentoo Hardened 4.8.3 p1.1, pie-0.5.9)

Hardware: GPU nvidia GTX 460

Reproducible: Always
```

**Comment by ProMcTagonist** [comment]
```
Info:
OS: Debian Jessie GNU/Linux x86_64
Kernel: 3.16.0-4
Xorg: 1:7.7+7
nouveau: 1:1.0.11-1
Qt: 5.3.2
DE/WM: Openbox 3.5.2-8

gcc: 4:4.9.1-5
gcc-4.8: 4.8.3-13
gcc-4.9: 4.9.1-19

Hardware: NVIDIA Corporation G84GLM [Quadro FX 570M] (rev a1)

Reproducible: Always
```

**Comment by agilob** [comment]
```
Info:
OS: Arch
Kernel: 3.18.6-1
Xorg: 1.17.1
nouveau: 1.0.11
Qt: 5.4.1
DE/WM: KDE4

Compiled with CLANG 3.5.1

Hardware: Intel something
Reproducible: Always
```

It's a problem with repaining FriendWidget on mouse-over. Title of the group is repainted after changing it, the rest of the widget is not and colors stay the same as they were on 'mouse-over'. It was reported once and fixed but only for contact.

How to reproduce:
1. Create new group
2. Select contact X
3. Right click on the group
4. Change title of the group
5. Chat with contact X should be active
6. ![](https://cloud.githubusercontent.com/assets/2544251/6546151/f565a798-c59f-11e4-884c-eb8815c203e9.png)

**Comment by agilob** [comment]
@dubslow Did I close it with commit or you manually?

**Comment by dubslow** [comment]
It says "closed this issue from a commit" with your commit under it. :)

GH's parsing is simplistic, all it saw as "closes #1330". And besides, zetok said it did indeed fix the issue.

**Comment by agilob** [comment]
Ok, but it says `dubslow closed this in b489e7f 3 hours ago`, I wasn't sure if I closed it or you did ;)

**Comment by dubslow** [comment]
We both did. GitHub closed it automatically when I, a maintainer, pushed your commit with its comment, to master. Since I decided to push to master, it says I closed the issue, but it was your commit comment that triggered the autoclose in the first place.
